src/api/entities/Portfolio/index.ts

```diff
@@ -212,6 +216,91 @@ export abstract class Portfolio extends Entity<UniqueIdentifiers, HumanReadable>
     return values(assetBalances);
   }
 
+  /**
+   * Retrieve the NFTs held in this portfolio
+   *
+   *  @param args.assets - array of NftCollection (or tickers) for which to fetch holdings (optional, all holdings are retrieved if not passed)
+   */
+  public async getNftsHeld(args?: {
```

Comment:
Also looks like we use `getHeldAssets` for a historical middleware query. I'll see if I can come up with something else.

Reply:
I am thinking to use "collections" and since NFT can be implied.

e.g.
```ts
public async getCollections(args?: {
    collections: (string | NftCollection)[];
  }): Promise<PortfolioCollection[]> {
```